android/app/src/main/java/cz/quanti/rocketapp/di/RocketModule.kt

```diff
@@ -4,11 +4,15 @@ import androidx.navigation.navOptions
 import cz.quanti.rocketapp.R
 import cz.quanti.rocketapp.presentation.RocketDetailViewModel
 import cz.quanti.rocketapp.presentation.RocketListViewModel
+import cz.quanti.rocketropository.di.rocketRepositoryModule
 import org.koin.androidx.viewmodel.dsl.viewModelOf
 import org.koin.dsl.module
 
 val rocketModule = module {
+    includes(rocketRepositoryModule)
```

Comment:
Je to dobry napad, udelat chain modulu? Nebo se v praxi casto stane, ze jsou dva moduly, poskytujici ruzne implementace stejnych interfacu?